### PR TITLE
Apply composer update to 4.1-core

### PR DESCRIPTION
**Pull request by nanasess**
## 概要(Overview・Refs Issue)

composer update を実行

<details>

  - Removing ocramius/proxy-manager (2.1.1)
  - Removing zendframework/zend-code (3.4.1)
  - Removing zendframework/zend-eventmanager (3.2.1)
  - Upgrading behat/gherkin (v4.6.2 => v4.7.3)
  - Upgrading codeception/phpunit-wrapper (6.8.2 => 6.8.3)
  - Upgrading composer/ca-bundle (1.2.8 => 1.2.9)
  - Upgrading composer/composer (2.0.2 => 2.0.12)
  - Upgrading composer/package-versions-deprecated (1.11.99 => 1.11.99.1)
  - Upgrading composer/semver (3.2.1 => 3.2.4)
  - Upgrading composer/spdx-licenses (1.5.4 => 1.5.5)
  - Upgrading composer/xdebug-handler (1.4.3 => 1.4.6)
  - Upgrading doctrine/annotations (1.10.4 => 1.12.1)
  - Upgrading doctrine/dbal (v2.9.3 => 2.13.0)
  - Locking doctrine/deprecations (v0.5.3)
  - Upgrading doctrine/doctrine-bundle (1.12.10 => 1.12.13)
  - Upgrading doctrine/doctrine-fixtures-bundle (3.3.2 => 3.4.0)
  - Upgrading doctrine/instantiator (1.3.1 => 1.4.0)
  - Upgrading doctrine/orm (2.7.4 => 2.7.5)
  - Upgrading doctrine/reflection (1.2.1 => 1.2.2)
  - Upgrading egulias/email-validator (2.1.22 => 2.1.25)
  - Upgrading friendsofphp/php-cs-fixer (v2.18.2 => v2.18.4)
  - Locking friendsofphp/proxy-manager-lts (v1.0.3)
  - Upgrading fzaninotto/faker (v1.9.1 => v1.9.2)
  - Upgrading guzzlehttp/promises (1.4.0 => 1.4.1)
  - Upgrading guzzlehttp/psr7 (1.7.0 => 1.8.1)
  - Locking laminas/laminas-code (3.4.1)
  - Locking laminas/laminas-eventmanager (3.2.1)
  - Locking laminas/laminas-zendframework-bridge (1.1.1)
  - Upgrading mobiledetect/mobiledetectlib (2.8.34 => 2.8.37)
  - Upgrading monolog/monolog (1.25.5 => 1.26.0)
  - Upgrading myclabs/deep-copy (1.10.1 => 1.10.2)
  - Upgrading nikic/php-parser (v4.10.2 => v4.10.4)
  - Upgrading php-coveralls/php-coveralls (v2.4.1 => v2.4.3)
  - Upgrading php-cs-fixer/diff (v1.3.0 => v1.3.1)
  - Upgrading sebastian/code-unit-reverse-lookup (1.0.1 => 1.0.2)
  - Upgrading sebastian/exporter (3.1.2 => 3.1.3)
  - Upgrading sebastian/object-enumerator (3.0.3 => 3.0.4)
  - Upgrading sebastian/object-reflector (1.1.1 => 1.1.2)
  - Upgrading sebastian/recursion-context (3.0.0 => 3.0.1)
  - Upgrading seld/jsonlint (1.8.2 => 1.8.3)
  - Upgrading setasign/fpdi (v2.3.4 => v2.3.6)
  - Upgrading swiftmailer/swiftmailer (v6.2.3 => v6.2.7)
  - Upgrading symfony/asset (v4.4.18 => v4.4.20)
  - Upgrading symfony/browser-kit (v4.4.18 => v4.4.20)
  - Upgrading symfony/cache (v4.4.18 => v4.4.21)
  - Upgrading symfony/config (v4.4.18 => v4.4.20)
  - Upgrading symfony/console (v4.4.18 => v4.4.21)
  - Upgrading symfony/css-selector (v4.4.18 => v4.4.20)
  - Upgrading symfony/debug (v4.4.18 => v4.4.20)
  - Upgrading symfony/debug-bundle (v4.4.18 => v4.4.20)
  - Upgrading symfony/dependency-injection (v4.4.18 => v4.4.21)
  - Upgrading symfony/doctrine-bridge (v4.4.18 => v4.4.21)
  - Upgrading symfony/dom-crawler (v4.4.18 => v4.4.20)
  - Upgrading symfony/dotenv (v4.4.18 => v4.4.20)
  - Upgrading symfony/error-handler (v4.4.18 => v4.4.21)
  - Upgrading symfony/event-dispatcher (v4.4.18 => v4.4.20)
  - Upgrading symfony/expression-language (v4.4.18 => v4.4.20)
  - Upgrading symfony/filesystem (v4.4.18 => v4.4.21)
  - Upgrading symfony/finder (v4.4.18 => v4.4.20)
  - Upgrading symfony/flex (v1.11.0 => v1.12.2)
  - Upgrading symfony/form (v4.4.18 => v4.4.21)
  - Upgrading symfony/framework-bundle (v4.4.18 => v4.4.21)
  - Upgrading symfony/http-foundation (v4.4.18 => v4.4.20)
  - Upgrading symfony/http-kernel (v4.4.18 => v4.4.21)
  - Upgrading symfony/inflector (v4.4.18 => v4.4.21)
  - Upgrading symfony/intl (v4.4.18 => v4.4.20)
  - Upgrading symfony/maker-bundle (v1.26.1 => v1.30.2)
  - Upgrading symfony/mime (v4.4.18 => v4.4.21)
  - Upgrading symfony/monolog-bridge (v4.4.18 => v4.4.21)
  - Upgrading symfony/monolog-bundle (v3.6.0 => v3.7.0)
  - Upgrading symfony/options-resolver (v4.4.18 => v4.4.20)
  - Upgrading symfony/phpunit-bridge (v4.4.18 => v4.4.21)
  - Upgrading symfony/polyfill-ctype (v1.20.0 => v1.22.1)
  - Upgrading symfony/polyfill-iconv (v1.20.0 => v1.22.1)
  - Upgrading symfony/polyfill-intl-icu (v1.20.0 => v1.22.1)
  - Upgrading symfony/polyfill-intl-idn (v1.20.0 => v1.22.1)
  - Upgrading symfony/polyfill-intl-normalizer (v1.20.0 => v1.22.1)
  - Upgrading symfony/polyfill-mbstring (v1.20.0 => v1.22.1)
  - Upgrading symfony/polyfill-php72 (v1.20.0 => v1.22.1)
  - Upgrading symfony/polyfill-php73 (v1.20.0 => v1.22.1)
  - Upgrading symfony/polyfill-php80 (v1.20.0 => v1.22.1)
  - Upgrading symfony/process (v4.4.18 => v4.4.20)
  - Upgrading symfony/property-access (v4.4.18 => v4.4.20)
  - Upgrading symfony/proxy-manager-bridge (v4.4.18 => v4.4.20)
  - Upgrading symfony/routing (v4.4.18 => v4.4.20)
  - Upgrading symfony/security (v4.4.18 => v4.4.21)
  - Upgrading symfony/security-bundle (v4.4.18 => v4.4.21)
  - Upgrading symfony/serializer (v4.4.18 => v4.4.20)
  - Upgrading symfony/stopwatch (v4.4.18 => v4.4.20)
  - Upgrading symfony/swiftmailer-bundle (v3.5.1 => v3.5.2)
  - Upgrading symfony/templating (v4.4.18 => v4.4.20)
  - Upgrading symfony/translation (v4.4.18 => v4.4.21)
  - Upgrading symfony/twig-bridge (v4.4.18 => v4.4.21)
  - Upgrading symfony/twig-bundle (v4.4.18 => v4.4.20)
  - Upgrading symfony/validator (v4.4.18 => v4.4.21)
  - Upgrading symfony/var-dumper (v4.4.18 => v4.4.21)
  - Upgrading symfony/var-exporter (v4.4.18 => v4.4.20)
  - Upgrading symfony/web-profiler-bundle (v4.4.18 => v4.4.21)
  - Upgrading symfony/web-server-bundle (v4.4.18 => v4.4.21)
  - Upgrading symfony/workflow (v4.4.18 => v4.4.20)
  - Upgrading symfony/yaml (v4.4.18 => v4.4.21)
  - Upgrading tecnickcom/tcpdf (6.3.5 => 6.4.1)

</details>

## 実装に関する補足(Appendix)

以下が原因でテストが落ちるため、 テストケースを修正

- checkboxを配列で渡すように修正。 Symfony4.4.20 でチェックが厳密になったためエラーになっていた
  - https://github.com/symfony/symfony/pull/39659
- Bcc で送信されるメールは1通とし、 RCPT To の宛先アドレスもカウントするように修正。 SwiftMailer6.2.6 で、 Bcc は1通のメールで送信されるようになったためエラーになっていた
   - https://github.com/swiftmailer/swiftmailer/pull/1134

## テスト（Test)

GitHub Actions が通るのを確認

## マイナーバージョン互換性保持のための制限事項チェックリスト

- [ ] 既存機能の仕様変更
- [x] フックポイントの呼び出しタイミングの変更
- [x] フックポイントのパラメータの削除・データ型の変更
- [x] twigファイルに渡しているパラメータの削除・データ型の変更
- [x] Serviceクラスの公開関数の、引数の削除・データ型の変更
- [x] 入出力ファイル(CSVなど)のフォーマット変更

## レビュワー確認項目

- [x] 動作確認
- [x] コードレビュー
- [x] E2E/Unit テスト確認(テストの追加・変更が必要かどうか)
- [x] 互換性が保持されているか
- [x] セキュリティ上の問題がないか

### CODE REVIEW

**Comment by nanasess** [comment]
間違えて Draft はずして元に戻せなくなってしまいました😥

**Comment by okazy** [comment]
ありがとうございます。
Draftにできました。
こちらにボタンがありました。
権限の問題でしょうか。

image

**Comment by nanasess** [comment]
@okazy ありがとうございます。権限の問題な気がします😥

**Comment by okazy** [comment]
SwiftMailer6.2.6 の懸念点として例えば

- TO: 個人のアドレス@google.com
- BCC: backup@google.com

の時に受け取り側で１通に見えてしまい、テスト時に誤解を招く必要がある。

内部的に１通しか送らないので、BCCの送信エラーでTOも送られない可能性がある。（メールサーバの実装による）

懸念点はありますが、RFCとしては正しい。
またマイナーバージョンでの変更なので、この変更を取り込めないと他のマイナーバージョンの変更も取り込めなくなってし今います。

以上より、懸念点はありますが、 EC-CUBE 4.1 としてはこちらを取り込む方針で行きたいと思います。

File: composer.json [hunk 1]
```diff
@@ -169,7 +169,6 @@
         ]
     },
     "conflict": {
-        "symfony/debug": "3.4.4",
```
Comment (okazy) [inline composer.json:172]: こちらの対策していただいていました。Symfony4.4では不要。
https://github.com/EC-CUBE/ec-cube/pull/2807#issuecomment-361546701

**Review by okazy — APPROVED** [review]
ありがとうございます。
修正内容問題ありません。
テストも通っていて動作OKです。

**Comment by okazy** [comment]
ありがとうございます。マージしました。

**Pull request merged**